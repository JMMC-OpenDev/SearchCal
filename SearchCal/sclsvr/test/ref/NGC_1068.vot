version https://git-lfs.github.com/spec/v1
oid sha256:77a5921105ab601e21cc86747371c34f06c63d1f2f824dbd998c89974c134c3a
size 220994
