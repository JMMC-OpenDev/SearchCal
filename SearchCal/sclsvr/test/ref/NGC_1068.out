version https://git-lfs.github.com/spec/v1
oid sha256:ebb37421f9676838fbf039071c6e148e07e2783701ab24b5fc0871cb6ade7eae
size 349298
